eo.txt

```diff
@@ -2407,12 +2408,10 @@ client_packet(NPCMapInfo, Request)
 "MapInfo for characters and/or npcs"
 server_packet(MapInfo, Reply)
 {
-	char num_entities
-	break // omitted if not sending back characters
-	struct CharacterMapInfo characters[num_entities] // omitted if not sending back characters
-	struct NPCMapInfo npcs[] // reads to end
+	struct NearbyInfo nearby
```

Comment:
Not sure I agree with this.. The packet structure is identical but sending ItemMapInfo structs doesn't actually do anything in the client. The client seems to just ignore them

Reply:
I guess it's fine to use NearbyInfo for this. Maybe we just need a comment to say "Items are ignored"

Reply:
Hmm. Are you certain that the items are ignored?
Sausage seemed pretty convinced that the entirety of the packet data from both `PLAYERS_AGREE` and `APPEAR_REPLY` gets passed to the same `Handle_NearbyInfo` function in the client.

Reply:
My bad - ItemMapInfo should use TinyCoords not Coords. It does work